Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -139,8 +139,11 @@ void HiresTexture::Prefetch()
 {
 	Common::SetCurrentThreadName("Prefetcher");
 
-	size_t size_sum = 0;
-	size_t max_mem = MemPhysical() / 2;
+	size_t size_sum = 0;	
+	size_t sys_mem = MemPhysical();
+	size_t recommended_min_mem = 2 * (1024.0 * 1024.0);
```

Comment:
Does change to:
size_t recommended_min_mem = 2.0 * (1024.0 * 1024.0);

Help with Windows compiler warning?
'initializing' : conversion from 'double' to 'size_t', possible loss of data